test/elixir/test/basics_test.exs

```diff
@@ -58,6 +58,64 @@ defmodule BasicsTest do
     assert context[:db_name] in Couch.get("/_all_dbs").body, "Db name in _all_dbs"
   end
 
+  @tag :with_db
+  test "Database name with '+' should encode to '+'", _context do
+    set_config({"chttpd", "encode_plus", "false"})
+
+    random_number = :rand.uniform(16_000_000)
```

Comment:
You don't need  to use randomness here. The `with_db` annotation would create a random db  for you (see https://github.com/apache/couchdb/blob/main/test/elixir/lib/couch/db_test.ex#L27:L28)

Reply:
I have removed the `:with_db` tag from two of the tests.

Reply:
The [`random_db_name/1`](https://github.com/apache/couchdb/blob/main/test/elixir/lib/couch/db_test.ex#L79) is injected into the  scope of each test module (the module which has `use CouchTestCase` stanza).

Therefore you can use `db_name = random_db_name("random+test+db")`. However I  don't have strong opinion about it.